simphony/io/h5_lattice.py

```diff
@@ -28,20 +29,22 @@ def __init__(self, group):
             raise ValueError("Lattice file layout has an incompatible version")
 
         self._group = group
-        self._type = group.lattice.attrs.type
-        self._base_vect = group.lattice.attrs.base_vect
+        self._prim_cell = PrimitiveCell(group.lattice.attrs.prim_cell[0],
+                                        group.lattice.attrs.prim_cell[1],
+                                        group.lattice.attrs.prim_cell[2])
+        self._prim_cell._bravais_lattice = BravaisLattice(
```

Comment:
Good idea. PrimitiveCell can be modified to include bravais_lattice parameter. It will also simplify some of the classmethods.